openfe_gromacs/protocols/gromacs_md/md_methods.py

```diff
@@ -178,11 +202,11 @@ def get_gro_filename(self) -> list[pathlib.Path]:
             pus[0].outputs["system_gro"]
             for pus in self.data.values()
             if "GromacsMDSetupUnit" in pus[0].source_key
-        ]
+        ][0]
```

Comment:
Are these singular / you take the first element because the expectation is that the input gro file should be the same for all repeat?

If so:
1. Could you write a comment about it somewhere?
2. Could you update the docstring to not say "llist of paths" for here and I think also `get_top_filename`?

Reply:
Yes, only a single filepath will be returned as the SetupUnit is only run once, even with multiple repeats. I updated the docstring to clarify this.

Reply:
> the SetupUnit is only run once

Not to be fixed here, and I'll have to think about this one a bit more, I'm not sure that this doesn't break the model for gufe generations (i.e. extensions). Something to keep in mind in the future.